src/main/java/internet/market/Main.java

```diff
@@ -0,0 +1,19 @@
+package internet.market;
+
+import internet.market.dao.jdbc.ProductDaoJdbcImpl;
+import internet.market.model.Product;
+
+public class Main {
+
+    public static void main(String[] args) {
+        ProductDaoJdbcImpl productDaoJdbc = new ProductDaoJdbcImpl();
+        Product product1 = new Product("mediator", 100);
+        productDaoJdbc.create(product1);
+        System.out.println(productDaoJdbc.get(1L));
+        Product product2 = new Product("aF", 234);
+        product2.setId(1L);
```

Comment:
no. DB should be responsible for id creating

Reply:
done